ui/app/(ai)/analyst/(tools)/providers.ts

```diff
@@ -0,0 +1,35 @@
+import { tool } from "@langchain/core/tools";
+
+import { getProvider, getProviders } from "@/actions/providers";
+import { getProviderSchema, getProvidersSchema } from "@/types/lighthouse";
+
+export const getProvidersTool = tool(
```

Comment:
Following up on this comment: https://github.com/prowler-cloud/prowler/pull/7508#discussion_r2052086824 

Could we please use the `/app` folder only for routing purposes?

Thank you!

Reply:
Moved all tools code to `/lib/lighthouse/tools/` directory